app/jobs/stockit_sync_orders_package_job.rb

```diff
@@ -0,0 +1,22 @@
+class StockitSyncOrdersPackageJob < ActiveJob::Base
+  queue_as :stockit_sync_orders_package_updates
+
+  def perform(package_id, orders_package, operation)
```

Comment:
is `orders_package` a simple ruby object? i.e. not an active record obj? If so, could you rename it to something like `orders_package_id` just so it's clear. Job params should only be simple ruby objects as they need to be serialized in ActiveJob.